etc/docker/dev/docker-compose.yml

```diff
@@ -35,7 +36,7 @@ services:
 
   rucio:
     container_name: dev-rucio-1
-    image: "docker.io/${DOCKER_REPO:-rucio}/rucio-dev${RUCIO_TAG:+:release-${RUCIO_TAG}}"
+    image: "docker.io/${DOCKER_REPO:-rucio}/rucio-dev:${RUCIO_DEV_PREFIX:-}${RUCIO_TAG:-latest}"
```

Comment:
Following discussion - this is necessary for Podman compatibility, as there is no better way to allow for this kind of logic for now. We can simplify in future if we decide to remove the `release-` prefix from the dev rucio container.